CDCgov/prime-reportstream
# 6058 Added NK1 segment for FHIR generation

This PR fixes a problem with the Next of Kin information missing from the FHIR bundle.

Test Steps:
1. Download this sample HL7 file and rename the extension to `.hl7`: 
[sample_hl7.txt](https://github.com/CDCgov/prime-reportstream/files/9085039/sample_hl7.txt)
2. Convert the HL7 to FHIR by running the following PRIME CLI command: `./prime fhirdata --input-file "<location of HL7 file>" --output-file build/hca-fhir.json`
3. Verify that the Patient.contact resource has the next of kin data from the NK1 segment.

## Changes
- Added NK1 segment to Patient resource

## Checklist

### Testing

### Process

## Linked Issues
- Fixes #issue

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue

## Specific Security-related subjects a reviewer should pay specific attention to
- Does this PR introduce new endpoints?
    - new endpoint A
    - new endpoint B
- Does this PR include changes in authentication and/or authorization of existing endpoints?
- Does this change introduce new dependencies that need vetting?
- Does this change require changes to our infrastructure?
- Does logging contain sensitive data?
- Does this PR include or remove any sensitive information itself?

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?
